cisco_aci/metadata.csv

```diff
@@ -53,31 +29,7 @@ cisco_aci.tenant.egress_bytes.multicast.cum,gauge,,byte,,represents the sum tota
-cisco_aci.tenant.application.endpoint.ingress_bytes.drop.cum,gauge,,byte,,represents the Ingress drop bytes. The total sum of the values read. Note that this value continues through each interval without resetting to zero,0,cisco_aci,
-cisco_aci.tenant.application.endpoint.ingress_bytes.unicast.cum,gauge,,byte,,represents the Ingress Unicast Bytes. The total sum of the values read. Note that this value continues through each interval without resetting to zero,0,cisco_aci,
-cisco_aci.tenant.application.endpoint.ingress_bytes.multicast.cum,gauge,,byte,,represents the Ingress Multicast Bytes. The total sum of the values read. Note that this value continues through each interval without resetting to zero,0,cisco_aci,
-cisco_aci.tenant.application.endpoint.ingress_bytes.flood.cum,gauge,,byte,,represents the Ingress Flood Bytes. The total sum of the values read. Note that this value continues through each interval without resetting to zero,0,cisco_aci,
 cisco_aci.tenant.application.endpoint.fault_counter,gauge,,fault,,represents the warning Fault. This is the average value read by the counter during the collection interval. Note that this value resets to 0 at the beginning of each interval,0,cisco_aci,
-cisco_aci.tenant.application.endpoint.egress_bytes.unicast.rate,gauge,,byte,,represents the Egress Unicast Bytes. This is the rate of the counter during the collection interval. The rate is calculated by dividing the periodic value by the length of the collection interval. Note that this value resets to 0 at the beginning of each interval,0,cisco_aci,
-cisco_aci.tenant.application.endpoint.egress_bytes.multicast.rate,gauge,,byte,,represents the Egress Multicast Bytes. This is the rate of the counter during the collection interval. The rate is calculated by dividing the periodic value by the length of the collection interval. Note that this value resets to 0 at the beginning of each interval,0,cisco_aci,
-cisco_aci.tenant.application.endpoint.egress_bytes.drop.cum,gauge,,byte,,represents the most current statistics for Egress Drop Packets,0,cisco_aci,
-cisco_aci.tenant.application.endpoint.egress_bytes.unicast.cum,gauge,,byte,,represents the Egress Unicast Bytes. The total sum of the values read. Note that this value continues through each interval without resetting to zero,0,cisco_aci,
-cisco_aci.tenant.application.endpoint.egress_bytes.multicast.cum,gauge,,byte,,represents the Egress Multicast Bytes. The total sum of the values read. Note that this value continues through each interval without resetting to zero,0,cisco_aci,
-cisco_aci.tenant.application.endpoint.egress_bytes.flood.cum,gauge,,byte,,represents the Egress Flood Bytes. The total sum of the values read. Note that this value continues through each interval without resetting to zero,0,cisco_aci,
```

Comment:
Should we fix by changing dd_epg_metric = dd_metric.format(TENANT_PREFIX) => dd_epg_metric = dd_metric.format(ENDPOINT_GROUP_PREFIX) ?

https://github.com/DataDog/integrations-core/blob/d28d6b47e134c1561f1c3d22d65faf5397fd1d80/cisco_aci/datadog_checks/cisco_aci/aci_metrics.py#L125

Of course, this is a breaking change and would need the fix to be backward compatible.

Reply:
I don't think it's the right approach. We have 3 sets in the metadata right now: tenant, tenant.application, tenant.application.endpoint. Currently, tenant is published properly by the agent, the others 2 not. We can publish tenant.application.endpoint instead, but it would break people that fixed our dashboard already.

Either we do code change + metadata, or just metadata. I think just metadata makes more sense.